### PR TITLE
Pub 1002 - updates to start page & service name change

### PR DESCRIPTION
**Pull request by furnivall**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1002

### Change description ###
Amended serviceName (i.e. black banner text at the top), altered tests, created new front page, added link to our service.

image
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/assets/scss/home.scss [hunk 1]
```diff
@@ -0,0 +1,4105 @@
+// Home page css override
```
Comment (kurt-uk) [inline src/main/assets/scss/home.scss]: can we not just import this? thats if we actually need it

Comment (furnivall) [reply src/main/assets/scss/home.scss]: If anyone else reads this, we've decided to include a heavily pruned version of the CSS from the initial commit (4k lines down to ~180ish) as there were issues with importing the relevant gov.uk CSS file.

File: src/test/e2e/Helpers/Selectors.ts [hunk 2]
```diff
@@ -9,8 +9,9 @@ module.exports = {
   SearchInput: '#search-input',
 
   // Home selectors
-  StartNowButton: '.govuk-button--start',
-  MainHeader: 'h1.govuk-heading-xl',
+  StartNowButton: '.govuk-button--start', //unused currently, to be re-used if start page reverts.
```
Comment (kurt-uk) [inline src/test/e2e/Helpers/Selectors.ts:12]: id bin this off, if its not used, odds are were going to be sticking with this for now

File: src/test/e2e/pageobjects/Home.page.ts [hunk 3]
```diff
@@ -12,9 +12,16 @@ export class HomePage extends CommonPage {
     return $(helpers.MainHeader).getText();
   }
 
+  // Old function for previous start page screen - leaving in case of future reversion.
   async clickStartNowButton(): Promise<ViewOptionPage> {
     const button = await $(helpers.StartNowButton);
     button.click();
     return new ViewOptionPage();
   }
```
Comment (kurt-uk) [inline src/test/e2e/pageobjects/Home.page.ts:20]: yeet it outta there, if we need it we can revert

File: src/test/e2e/tests/specs.test.ts [hunk 4]
```diff
@@ -41,11 +41,19 @@ let caseEventGlossaryPage: CaseEventGlossaryPage;
...
-  it('should click on the \'Start now\' button and navigate to View Options page', async () => {
-    viewOptionPage = await homePage.clickStartNowButton();
+
+  // *** the below function was previously used to navigate from the old start page
+  // - leaving in to aid reversion if required ***
+  // it('should click on the \'Start now\' button and navigate to View Options page', async () => {
+  //   viewOptionPage = await homePage.clickStartNowButton();
+  //   expect(await viewOptionPage.getPageTitle()).toEqual('What would you like to view?');
+  // });
+
```
Comment (kurt-uk) [inline src/test/e2e/tests/specs.test.ts:54]: i think you know what im gunna say

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: .eslintignore [hunk 5]
```diff
@@ -12,3 +12,4 @@ jest.*config.js
 *.min.js
 /src/main/assets/js/jquery.js
 /src/main/assets/js/mojAll.js
+/src/main/assets/scss/home.scss
```
Comment (ChrisS1512) [inline .eslintignore:15]: With the file now being smaller, can we remove the exclusion + fix any linting issue if possible?

File: src/test/e2e/Helpers/Selectors.ts (hunk 2 above)
Comment (furnivall) [reply src/test/e2e/Helpers/Selectors.ts]: sorted.

File: src/test/e2e/pageobjects/Home.page.ts (hunk 3 above)
Comment (furnivall) [reply src/test/e2e/pageobjects/Home.page.ts]: sorted

File: src/test/e2e/tests/specs.test.ts (hunk 4 above)
Comment (furnivall) [reply src/test/e2e/tests/specs.test.ts]: Roger 👍

File: src/main/views/home.njk [hunk 6]
```diff
@@ -3,39 +3,928 @@
     {{ title }}
```
Comment (ChrisS1512) [inline src/main/views/home.njk:3]: Ok really minor, but any chance you could remove the extra spacing in here between a lot of the sections?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/views/home.njk (hunk 6 above)
Comment (furnivall) [reply src/main/views/home.njk]: sorted.

File: .eslintignore (hunk 5 above)
Comment (furnivall) [reply .eslintignore]: sorted.

**Review by kurt-uk — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**